README.md

```diff
@@ -35,6 +35,7 @@ Both generation pipelines are still in `a work in progress` state, and several i
 |  `YARP_ROBOT_NAME`   | Pipeline     | Notes                           |
 |:--------------------:|:------------:|:-------------------------------:|
 | `iCubDarmstadt01`    | simmechanics | v2.5 without backpack           |
+| `iCubErzelli02`      | simmechanics | v2.5   with backpack            |
```

Comment:
```suggestion
| `iCubErzelli02`      | simmechanics | v2.5.5         |
```

The correct version identified for v2.5 with backpack is v2.5.5, see https://icub-tech-iit.github.io/documentation/icub_versions/ . The other entries are outdated and probably could be updated in another issue.